tests/FileDetector.php

```diff
@@ -275,39 +275,40 @@ private static function IsEngineGodot( array $Files ) : bool
 		//1. A single .pck file named exactly "data.pck", and NO other pck files
 		//2. For every executable, a correspondingly named pck file, and no other pck files
 
-		$swapExtension = static fn( string $FileName, string $OldExtension, string $NewExtension ) : string => basename( $FileName, $OldExtension ) . $NewExtension;
+		$swapExtension = static fn( string $FilePath, string $OldExtension, string $NewExtension ) : string => preg_replace('/^.\//','',dirname($FilePath) . '/' . basename( $FilePath, $OldExtension ) . $NewExtension);
```

Comment:
What is this doing?

Reply:
basename removes the path
so i add it back with dirname but when a file is at the root, it returns '.'
issue is: files we got don't start with './' so i'm removing it with preg_replace to match with the files we have